modules/subscriptions/views.php

```diff
@@ -727,7 +727,9 @@ function jetpack_blog_subscriptions_init() {
 add_action( 'widgets_init', 'jetpack_blog_subscriptions_init' );
 
 function jetpack_register_subscriptions_block() {
-	jetpack_register_block( 'subscriptions' );
+	if ( class_exists( 'WP_Block_Type_Registry' ) && WP_Block_Type_Registry::get_instance()->is_registered( 'jetpack/subscriptions' ) ) {
```

Comment:
maybe I'm missing something, but should not we check that the block is not currently registered? 

```suggestion
	if ( class_exists( 'WP_Block_Type_Registry' ) && ! WP_Block_Type_Registry::get_instance()->is_registered( 'jetpack/subscriptions' ) ) {
```

Reply:
:man_facepalming: Thanks Miguel.